glide.yaml

```diff
@@ -25,38 +18,44 @@ import:
 - package: github.com/go-ini/ini
 - package: github.com/gorilla/mux
 - package: github.com/coreos/pkg/capnslog
-- package: github.com/kubernetes-incubator/external-storage
-  version: 999a5f33de0f6c51f7325f696ac738d8fc73ec65
-- package: k8s.io/client-go
-  version: 42a124578af9e61f5c6902fa7b6b2cb6538f17d2
-- package: k8s.io/apimachinery
-  version: 728b986edb9bd4bb8a022a8cc13573a97ecc2239
-- package: k8s.io/api
-  version: e24ed681f56165befbea50a24f7e9f1c079ac366
-- package: k8s.io/apiextensions-apiserver
-  version: 3eb4c5fefbe7ad866747f78c7e4994cb555170c6
 - package: github.com/prometheus/client_golang
   version: e7e903064f5e9eb5da98208bae10b475d4db0f8c
   subpackages:
   - prometheus
   - prometheus/promhttp
 - package: github.com/jbw976/go-ps
   version: 82859aed1b5dafbbe6ad4f3dac5404d2a742cf18
+# Kuberenetes dependencies. These must all be updated together
+# as a compatible set.
+- package: github.com/kubernetes-incubator/external-storage
+  version: 0944d3777135a29a532121db4a3282b58d3d3a3f
+- package: k8s.io/apiserver
+  version: 149fc2228647cea28b0670c240ec582e985e8eda
+- package: k8s.io/apiextensions-apiserver
+  version: be41f5093e2b05c7a0befe35b04b715eb325ab43
+- package: k8s.io/apimachinery
+  version: 728b986edb9bd4bb8a022a8cc13573a97ecc2239
+- package: k8s.io/client-go
+  version: 42a124578af9e61f5c6902fa7b6b2cb6538f17d2
+- package: k8s.io/api
+  version: e24ed681f56165befbea50a24f7e9f1c079ac366
+- package: k8s.io/kubernetes
+  version: df7f4b3526a580ef122aaeef61ab52842a60a88b
+#testImport: imports only used during testing go below
```

Comment:
you can still use testImport for test packages that dont conflict with `import`

Reply:
its best we avoid it all together. `dep` doesn't have it, and the glide folks don't seem to have a solution for it. see referenced issue.